packages/cli/src/Db.ts

```diff
@@ -179,6 +180,7 @@ export async function init(testConnectionOptions?: ConnectionOptions): Promise<v
 	collections.Variables = Container.get(VariablesRepository);
 	collections.WorkflowStatistics = Container.get(WorkflowStatisticsRepository);
 	collections.WorkflowTagMapping = Container.get(WorkflowTagMappingRepository);
+	collections.WorkflowHistory = Container.get(WorkflowHistoryRepository);
```

Comment:
we should avoid adding new repositories here, to help move away from `Db.collections`, and use dependency injection wherever possible.

Reply:
Maybe we can add at the top:

```ts
/**
* @important Do not add to these collections. Inject the repository as a dependency instead.
*/
```

Reply:
Great idea, added the comments, created the Workflow History Service and set it to become a service.